src/blocks/media-block/ImageBlock.new.test.tsx

```
@@ -0,0 +1,463 @@
+  const MockMotionImg = React.forwardRef<HTMLImageElement, MotionComponentProps & React.ImgHTMLAttributes<HTMLImageElement>>(
+    (props, ref) => {
+      const { children, alt = '', ...rest } = props;
+      return (
+        <img 
+          ref={ref} 
+          alt={alt}
+          data-testid={props['data-testid'] || 'motion-img'}
+          {...rest}
+        >
+          {children}
+        </img>
```

Biome (1.9.4)

[error] img is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)